### ISSUE TITLE
Terminübersicht und Widget defekt

### ISSUE THREAD

**Issue by marteb** [kre8tiv/urwahl3000#5]
Seit dem Update 2.1.4, mit der Umstellung von wpCalendar auf Kal3000, ist die Listenansicht der Termine und das Termine-Widget fehlerhaft, siehe http://www.gruene-rhein-pfalz.de/termine/

**Comment by modulbuero** [comment]
Hi, du meinst vermutlich die geografischen Daten direkt unter dem Datum, richtig? Die kannst du per CSS ausblenden mit der folgenden Definition:

`.widget_kal3000_termine_liste_widget` span.termin_ort_kurz { 
	display: none; 
}`

Hilft dir das schon weiter oder habe ich etwas falsch verstanden?

**Comment by marteb** [comment]
Hallo, nein, ich meine, dass unter den Terminen im Seiteninhalt links nicht mehr ein Auszug aus dem Termin erscheint und bei den Terminen im Widget (Seitenleiste rechts) nun dieser Auszug und noch mehr ausgegeben wird.

**Comment by marteb** [comment]
Ich merke gerade, dass der Fehler rechts im Widget wohl tatsächlich durch alte Styles im Browser-Cache zustande kam. In der Listenansicht der Termine scheint allerdings im Vergleich zu Vorher der Auszug aus dem Termin zu fehlen.

<img width="864" alt="bildschirmfoto_2018-01-14_um_11_39_08" src="https://user-images.githubusercontent.com/1926178/34915098-d7fa380c-f91f-11e7-910f-cbe9c8371569.png">

**Comment by modulbuero** [comment]
Mh, der Cache hätte sich eigentlich durch die stets neue Versionsnummer selbst leeren sollen, aber zugegebenermaßen werden die Browser bei dem Thema immer hartnäckiger. 

Die Darstellungsvarianten des Widgets sind mir auch schon aufgefallen. Bisher scheint das immer über CSS gelaufen zu sein – sprich es ist der gesamte Content vorhanden, wird aber dann je nach Anwendung ausgeblendet. Ich überlege jetzt, ob wir stattdessen das Widget konfigurierbar machen und einfach sagen: Hier bitte mit Beschreibungstext, hier bitte ohne. Wäre das für dich eine Hilfe, @marteb?

**Comment by marteb** [comment]
Hallo, ja, ein Lösung über konfigurierbares Widget wäre sicherlich super! Das derzeit noch bestehende Problem gibt es bei der Darstellung im Inhaltsbereich per Shortcode [wpcalendar]. Oder gibt es da mittlerweile einen neuen Shortcode?

**Comment by modulbuero** [comment]
Nein, der Shortcode ist der gleiche. Dort angezeigt wird das "Auszug"-Feld aus dem Backend, probier' das vielleicht mal aus. 

Nichtsdestotrotz setze ich die Weiterentwicklung von Widget und Shortcode auf meine Liste und werde das irgendwann in den nächsten Wochen nachrüsten.
